src/Bridge/Symfony/Bundle/Resources/config/graphql.xml

```diff
@@ -77,6 +77,10 @@
             <argument type="service" id="api_platform.item_data_provider" on-invalid="ignore" />
             <argument>%api_platform.allow_plain_identifiers%</argument>
 
+            <call method="setItemNormalizer">
```

Comment:
We don't use setter injection in API Platform. Can you inject it directly in the constructor please?

Reply:
I've done it this way because the constructor of `AbstractItemNormalizer` has a lot of dependencies and I would need to copy it in this normalizer. But I can do it if you prefer.

Reply:
WDYT @dunglas?

Reply:
Yes I prefer to copy them. Please :)